dockerize.nix

```diff
@@ -11,7 +11,8 @@ dockerTools.buildImage {
   ${dockerTools.shadowSetup}
   mkdir -p /root/.ssh
   '';
-  contents = [marge pkgs.bash pkgs.coreutils pkgs.openssh];
+  # gnu sed and grep are needed for git filter-branch; in other words pkgs.git seems incomplete
+  contents = [marge pkgs.bash pkgs.coreutils pkgs.openssh pkgs.gnused pkgs.gnugrep];
   config = {
```

Comment:
@aschmolck Adding `sed` and `grep` here is pointless, right? If so, d8299af86462b7efe could be dropped.